Papr/Utils/Extentions/Date+Extentions.swift

```diff
@@ -39,9 +39,9 @@ extension Date {
             return "\(Int(self.since(Date(), in: .hour).rounded()))h"
         } else if roundedDate >= 24 * 60.0 && roundedDate < 7 * 24 * 60 {
             return "\(Int(self.since(Date(), in: .day).rounded()))d"
-        } else if roundedDate >= 7 * 24 * 60.0 && roundedDate < 30 * 24 * 60 {
+        } else if roundedDate >= 7 * 24 * 60.0{
             let dateformatter = DateFormatter()
-            dateformatter.dateFormat = "MM/dd/yy"
+            dateformatter.dateFormat = "dd/mm/yy"
```

Comment:
Should be `MM` not `mm`. My mistake that I suggested `mm`.

Reply:
👍